specification/evaluation-context/evaluation-context.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "id": "Requirement 3.1",
+        "clean id": "requirement_3_1",
+        "content": "The `evaluation context` structure MUST define a required `targeting key` field of type string, identifying the subject of the flag evaluation.",
+        "RFC 2119 keyword": "MUST",
+        "children": []
+    },
+    {
+        "id": "Requirement 3.2",
+        "clean id": "requirement_3_2",
+        "content": "The evaluation context MUST define the following optional fields: `email` (string), `first name` (string), `last name`(string), `name`(string), `client ip`(string), `tz`(string), `locale`(string), `date`(string | date), `environment`(string), `application`(string), and `version`(string).",
+        "RFC 2119 keyword": "MUST",
+        "children": []
```

Comment:
It would be great if the children section could contain the optional fields.

Reply:
`children` is for nested conditional requirements. Can you clarify what you mean?

Reply:
I was just saying that it may be easier to read if each field was a separate child requirement.

Reply:
I've separated all the fields into their own requirement, but kept the table for an overview as well.